lib/user_session_management.rb

```diff
@@ -58,7 +64,6 @@ def authenticate_admin!
   alias_method :admin_authentication!, :authenticate_admin!
 
   def set_login_state(username_or_email: nil, matching_user_ids: nil, names: nil, providers: nil)
-    clear_signup_state
```

Comment:
we now don't delete this because the `signup_state` is now holding the signed data?

Reply:
yes, and it's needed if the user switches from sign-in to sign-up by clicking the link at bottom of sign-in modal.

Previously this was cleared here and then the sign-up wouldn't be trusted